Automattic/woocommerce-payments
# Registering WC Pay with Nav clientside

While reviewing https://github.com/Automattic/woocommerce-payments/pull/1067 I noticed a big flaw in how we register items on the client. We were trying to use the `path` as a unique identifier which failed when items were being registered to a `parent` because a mismatch in `id`. More on that in https://github.com/woocommerce/woocommerce-admin/pull/5773

This PR updates to the changes introduced in https://github.com/woocommerce/woocommerce-admin/pull/5773

#### Testing instructions

Use the WC Admin branch in https://github.com/woocommerce/woocommerce-admin/pull/5773 to make sure links work correctly and do not incur a page refresh when they should not. Specifically, these issues should be resolved: https://github.com/Automattic/woocommerce-payments/pull/1067#issuecomment-735455853
